MAUI/Popup/popup-positioning.md

```diff
@@ -264,4 +264,300 @@ public class ViewModel : INotifyPropertyChanged
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:sfDatagrid="clr-namespace:Syncfusion.Maui.DataGrid;assembly=Syncfusion.Maui.DataGrid"
+             xmlns:sfPopup="clr-namespace:Syncfusion.Maui.Popup;assembly=Syncfusion.Maui.Popup"
+             xmlns:local="clr-namespace:PopupMauiPositioning"
+             x:Class="PopupMauiPositioning.MainPage">
+  <ContentPage.BindingContext>
+    <local:ViewModel x:Name="viewModel" />
+  </ContentPage.BindingContext>  
+  <ContentPage.Content>
+    <Grid>
+      <sfDatagrid:SfDataGrid x:Name="dataGrid"
+                             ItemsSource="{Binding OrdersInfo}"
+                             CellTapped="OnDataGridCellTapped"    
+                             ColumnWidthMode="Fill">
+      </sfDatagrid:SfDataGrid>
+      <sfPopup:SfPopup x:Name="sfPopup" 
+                       HeaderTitle="Popup" 
+                       AutoSizeMode="Height"
+                       ShowCloseButton="True">
+        <sfPopup:SfPopup.ContentTemplate>
+          <DataTemplate>
+            <Label Text="A pop-up is a graphical user interface display area that suddenly appears in the foreground of the visual interface. Pop-up can be initiated by single or double tap or can simply be timed to occur. A pop-up window should be smaller than the background window or interface; otherwise, its a replacement interface."                               
+                   TextColor="Black" 
+                   FontSize="14" 
+                   FontFamily="Roboto" 
+                   Padding="0,0,0,24" />
+          </DataTemplate>
+        </sfPopup:SfPopup.ContentTemplate>
+      </sfPopup:SfPopup>
+    </Grid>
+  </ContentPage.Content> 
+</ContentPage>
+{% endhighlight %}
+{% highlight c# tabtitle="C#" hl_lines="17" %} 
+namespace PopupMauiPositioning;
+using Syncfusion.Maui.DataGrid;
+using Syncfusion.Maui.Popup;
+
+public partial class MainPage : ContentPage
+{
+  SfDataGrid dataGrid;
+  ViewModel viewModel;
+  SfPopup sfPopup;
+
+  public MainPage()
+  {
+    InitializeComponent();
+    dataGrid = new SfDataGrid();
+    viewModel = new ViewModel();
+    dataGrid.ItemsSource = viewModel.OrdersInfo;
+    dataGrid.CellTapped += OnDataGridCellTapped;
+    dataGrid.ColumnWidthMode = ColumnWidthMode.Fill;
+
+    sfPopup = new SfPopup();
+    sfPopup.HeaderTitle = "Popup";
+    sfPopup.ShowCloseButton = true;
+    sfPopup.AutoSizeMode = PopupAutoSizeMode.Height;
+    sfPopup.ContentTemplate = new DataTemplate(() =>
+    {
+        var label = new Label();
+        label.Text = "A pop-up is a graphical user interface display area that suddenly appears in the foreground of the visual interface. Pop-up can be initiated by single or double tap or can simply be timed to occur. A pop-up window should be smaller than the background window or interface; otherwise, its a replacement interface.";

```

Comment:
label.Text = "A pop-up is a graphical user interface display area that suddenly appears in the foreground of the visual interface. The pop-up can be initiated by a single or double tap or can be timed to occur. A pop-up window should be smaller than the background window or interface; otherwise, it's a replacement interface.";